### ISSUE TITLE
"Leave a comment" text area should have a limit when minimized 

### ISSUE THREAD

**Issue by softvision-oana-arbuzov** [webcompat/webcompat.com#2166]
URL: https://staging.webcompat.com/issues/1378
Firefox Nightly Mobile 60.0a1 (2018-02-18), 
Operating System: Google Pixel (Android 8.0.0) - 1080 x 1920 pixels (~441 ppi pixel density)

**Prerequisites:**
1. Github account available and signed in.

**Steps to Reproduce:**
1. Navigate to https://staging.webcompat.com/issues/1378
2. Scroll down the page to "Comment" section.
3. Minimize the "Leave a comment" text area.
4. Type in the area and observe text.

**Expected Behavior:**
Text is fully visible. "Leave a comment" text area has a limit when minimized (1-2 rows).

**Actual Behavior:**
Text is not visible.

**Note**
1. Also reproducible on Desktop.
2. Affected area:
```html
<textarea id="newComment" class="form-field text-field light js-Comment-text mousetrap" rows="2" placeholder="Leave a comment" style="height: 134px;"></textarea>
```

Watchers:
@softvision-sergiulogigan
@softvision-oana-arbuzov

![screenshot-1519054991549](https://user-images.githubusercontent.com/12184325/36385838-79263b5c-159c-11e8-9cb4-9ae777a0c4d8.jpg)

![minimizedtextarea](https://user-images.githubusercontent.com/12184325/36385842-7af435ba-159c-11e8-9372-1803e5d4c138.gif)

**Comment by zoepage** [comment]
I think, this whole section needs work. :)

**Comment by zoepage** [comment]
merged in #2240
